packages/material/scss/tabstrip/_variables.scss

```diff
@@ -146,7 +146,7 @@ $kendo-tabstrip-item-selected-font-weight: null !default;
 
 /// The shadow of the focused TabStrip items.
 /// @group tabstrip
-$kendo-tabstrip-item-focus-shadow: $kendo-list-item-focus-shadow !default;
+$kendo-tabstrip-item-focus-shadow: inset 0 0 0 2px color-mix(in srgb, currentColor 20%, transparent) !default; // $kendo-list-item-focus-shadow
```

Comment:
Does the comment mean that the value should be replaced with the variable or is it just for reference. If it is for reference, maybe it can be removed?